content/en/error_tracking/frontend/browser.md

```diff
@@ -9,63 +9,310 @@ further_reading:
+
+##### Context and storage
+
+These parameters control how user context and application state are preserved across page navigation and browser sessions.
+
+| Parameter | Type | Default | Description |
+|-----------|------|---------|-------------|
+| `allowUntrustedEvents` | Boolean | `false` | Allow capture of [untrusted events][24], for example in automated UI tests. |
+| `storeContextsAcrossPages` | Boolean | `false` | Store global context and user context in `localStorage` to preserve them along user navigation. See [Contexts life cycle][23]. |
+
+##### Logs Browser SDK compatibility options
+
+These parameters ensure compatibility when using both the RUM Browser SDK and Logs Browser SDK together. They must have matching configuration values across both SDKs to maintain consistent session tracking and data correlation.
+
+| Parameter | Type | Default | Description |
+|-----------|------|---------|-------------|
+| `allowFallbackToLocalStorage` | Boolean | `false` | **Deprecated** - Use `sessionPersistence` instead. |
+| `sessionPersistence` | `"cookie" \| "local-storage"` | `"cookie"` | Which storage strategy to use for persisting sessions. Can be either `cookie` or `local-storage`. |
+| `trackSessionAcrossSubdomains` | Boolean | `false` | Preserve the session across subdomains for the same site. |
+| `usePartitionedCrossSiteSessionCookie` | Boolean | `false` | Use a partitioned secure cross-site session cookie. This allows the Browser SDK to run when the site is loaded from another one (iframe). Implies `useSecureSessionCookie`. |
+| `useSecureSessionCookie` | Boolean | `false` | Use a secure session cookie. This disables events sent on insecure (non-HTTPS) connections. |
+
+### Step 4 - Deploy your application
+
+Deploy the changes to your application. After your deployment is live, Datadog collects events from your users' browsers.
+
+### Step 5: Upload source maps (optional but recommended)
+
+Upload your JavaScript source maps to access unminified stack traces. See [Upload JavaScript source maps][30].
+
+### Step 6: Visualize your data
+
+Now that you've completed the basic setup for Browser Error Tracking, your application is collecting browser errors and you can start monitoring and debugging issues in real-time.
+
+Visualize the [data collected][7] in [dashboards][8] or create a search query in Error Tracking.
+
+Until Datadog starts receiving data, your application appears as `pending` on the **Applications** page.
+
+## Advanced features (optional)
+
+### Link errors with your source code
+
+In addition to sending source maps, the [Datadog CLI][25] reports Git information such as the commit hash, repository URL, and a list of tracked file paths in the code repository.
+
+Error Tracking can use this information to correlate errors with your source code, allowing you to pivot from any stack trace frame to the related line of code in [GitHub][26], [GitLab][27] and [Bitbucket][28].
```

Comment:
I think some of the links are off by 1. the GitHub is 25, GitLab 26, and Bitbucket 27